servers/quarkus-common/src/main/java/org/projectnessie/quarkus/reflect/ModelReflections.java

```diff
@@ -26,5 +26,11 @@
  * <p>In particular, this list is about Nessie model classes.
  */
 @RegisterForReflection(
-    targets = {NessieError.class, CELUtil.OperationForCel.class, CELUtil.ContentForCel.class})
+    targets = {
+      NessieError.class,
+      CELUtil.KeyedEntityForCel.class,
```

Comment:
just wondering:
`KeyedEntityForCel` is also new here, but mentioning int he commit msg is not important? :sweat_smile:

Reply:
It is. I just mentioned the top think that was added in #4913. `KeyedEntityForCel` was a result of local refactoring, but it still has to be available for introspection.

Reply:
updated commit message for this.